vespa/package.py

```diff
@@ -247,7 +338,8 @@ def __repr__(self):
             repr(self.match),
             repr(self.weight),
             repr(self.bolding),
-            repr(self.summary)
+            repr(self.summary),
+            repr(self.struct_fields),
```

Comment:
I guess this PR conflicts with https://github.com/vespa-engine/pyvespa/pull/411 and https://github.com/vespa-engine/pyvespa/pull/412 and https://github.com/vespa-engine/pyvespa/pull/410 here?

Reply:
Yes, each of my PRs (that the Field class will conflict upon merging either of them), in any case I think we should deal with the [amount of constructors issue](https://github.com/vespa-engine/pyvespa/pull/412#discussion_r1066205019) before merging any of them.